docs/api/synchronization_library/atomic.md

```diff
@@ -1,56 +1,75 @@
+---
+grand_parent: API
+parent: Synchronization Library
+nav_order: 1
+---
+
 # `<cuda/std/atomic>`, `<cuda/atomic>`
 
-## Extensions in the `cuda::` namespace
```

Comment:
Why are we dropping this part of the heading? The prose below doesn't seem to make it explicit that this only applies in `cuda::`, and the examples only hint at it.

Reply:
I just don't like it being so prosey. All the other section titles are short and succinct.

Reply:
It needs to go into the actual prose then.